src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -259,9 +259,12 @@ void mongoInit
   // "If you call multiple ensureIndex() methods with the same index specification at the same time,
   // only the first operation will succeed, all other operations will have no effect."
   //
-  ensureLocationIndex("");
-  ensureDateExpirationIndex("");
-  if (mtenant)
+  if (!mtenant)
+  {
+    ensureLocationIndex("");
+    ensureDateExpirationIndex("");
+  }
```

Comment:
```suggestion
  if (!mtenant)
  {
    // To avoid creating and empty 'orion' database when -multiservice is in use
    ensureLocationIndex("");
    ensureDateExpirationIndex("");
  }
```

I think this way it gets clearer why this guard is done.

Reply:
Updated file